cmd/start.go

```diff
@@ -210,6 +211,9 @@ func init() {
 		}
 	}
 
+	// Gateway Address
+	startCmd.Flags().StringVar(&startCmdArgs.Network.GatewayAddress, "gateway-address", "192.168.5.2", "gateway address")
```

Comment:
You can leverage ip address flag type to avoid manual validation later.

```suggestion
	startCmd.Flags().IPVar(&startCmdArgs.Network.GatewayAddress, "gateway-address", net.ParseIP("192.168.5.2"), "gateway address")
```